src/basic-samples/GuaranteedSubscriber.js

```diff
@@ -0,0 +1,254 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Solace Web Messaging API for JavaScript
+ * Receiving Guaranteed messages on a Topic tutorial - Guaranteed Subscriber
+ * Demonstrates receiving persistent messages on a topic
+ */
+
+/*jslint es6 browser devel:true*/
+/*global solace*/
+
+var GuaranteedSubscriber = function (solaceModule, queueName, topicName ) {
+    'use strict';
+    const API = 'js';
+    const TOPIC_PREFIX = "solace/samples/";  // used as the topic "root"
```

Comment:
vars `API ` and `TOPIC_PREFIX` are not used since you are passing `topicName` in the GuaranteedSubscriber function

Reply:
Once the topcName is gone, these related items are gone as well. Removed!